test/kuttl/tests/change_neutron_config/03-change_neutron_debug_config.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: neutron.openstack.org/v1beta1
+kind: NeutronAPI
+metadata:
+  name: neutron
+  namespace: openstack
```

Comment:
the namespace line should be removed, since we run the kuttl tests in a different namespace

Reply:
fixed, thank you.